src/spikeinterface/curation/curation_tools.py

```diff
@@ -133,3 +133,39 @@ def find_duplicated_spikes(
         return _find_duplicated_spikes_keep_last_iterative(spike_train.astype(np.int64), censored_period)
     else:
         raise ValueError(f"Method '{method}' isn't a valid method for find_duplicated_spikes. Use one of {_methods}")
+
+
+def get_new_unit_ids_for_merges(sorting, units_to_merge, new_unit_ids):
```

Comment:
thinking more about this: if left here, importing this will import numba and jit compile the numba functions. 

Maybe worth moving this function too to sorting_tools? It's general enough

Reply:
Done